src/iframe-attributes.js

```diff
@@ -57,6 +60,14 @@ export function getContextMetadata(
       href: locationHref,
     },
     startTime,
+    tagName: element.tagName,
+    mode: getModeObject(),
+    canary: isCanary(parentWindow),
+    hidden: !viewer.isVisible(),
+    initialIntersection: element.getIntersectionChangeEntry ?
```

Comment:
No, I was referring to the <amp-a4a> element, not the <amp-ad> element. The <amp-a4a> element does not have getIntersectionChangeEntry defined on it.